modules/tasks/tasks/src/test/java/org/motechproject/tasks/service/TaskTriggerHandlerTest.java

```diff
@@ -188,7 +198,7 @@ public void shouldNotRegisterHandler() {
 
         when(taskService.getAllTasks()).thenReturn(new ArrayList<Task>());
 
-        new TaskTriggerHandler(taskService, null, eventListenerRegistryService, null, taskActionExecutor, null);
```

Comment:
We need a test in this class that tests how the trigger handler treats scheduler jobs